### PR TITLE
Splash and Main Screen

### LINKED ISSUE #14: Create splash screen and initial screen
## REQUIREMENTS / DESCRIPTION

Create a splash screen for loading the assets and an initial screen to choose between login or sign up using the components made before.

## DESIGN

image

### PR DESCRIPTION
**Pull request by FebernalGober**
Main screen screens were created

In Splash, code was used directly instead of some library

for the main screen the reusable components LogoRFC and BasicButton were used

### CODE REVIEW

File: renta_facil_colima/App.jsx
Comment (BrandonVG) [inline renta_facil_colima/App.jsx:1]: Restore the file's content

**Review by BrandonVG — CHANGES_REQUESTED** [review]

**Review by Axw3ll — CHANGES_REQUESTED** [review]
Restore the file's content.The right thing to do is resolve conflicts.

**Review by Tole15 — CHANGES_REQUESTED** [review]
Restore App.jsx and delete comments on any code u have.
image

File: renta_facil_colima/App.jsx
Comment (DerekAyala) [inline renta_facil_colima/App.jsx:1]: Restore the file's content and use .tsx

File: renta_facil_colima/src/View/components/SplashScreen.jsx
Comment (DerekAyala) [inline renta_facil_colima/src/View/components/SplashScreen.jsx:1]: keep the styles on another file

**Review by DerekAyala — CHANGES_REQUESTED** [review]

**Review by BrandonVG — APPROVED** [review]

**Review by Axw3ll — APPROVED** [review]

**Review by Tole15 — APPROVED** [review]
Nice Fix!
image

**Review by DerekAyala — APPROVED** [review]

**Pull request merged**